app/controllers/api_controller.rb

```diff
@@ -102,6 +102,11 @@ def set_access_control_headers
   ID_ATTRS = %w(href id)
 
   #
+  # Requester types for authentication
+  #
+  REQUESTER_TYPES = {"ui" => :ui_token_ttl}
```

Comment:
This constant name doesn't make sense, or at least the value doesn't seem to apply based on the name.